src/sentry/templates/sentry/js-sdk-loader.ts

```diff
@@ -209,7 +209,11 @@ declare const __LOADER__IS_LAZY__: any;
         // Cleanup to allow garbage collection
         _window.sentryOnLoad = undefined;
       }
+    } catch (o_O) {
+      console.error(o_O);
```

Comment:
l: should we add an additional log that we fall back to the default init?

Reply:
Good question, I was thinking about this too. today we don't really have any logs there, as it _slightly)_ increases size, but maybe it would help 🤔  maybe just "Error during `sentryOnLoad`"?